notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
     "\n",
```

Comment:
<p><code>Here we can see that body_size is&nbsp;most&nbsp;correlated with time_to_merge,</code>  This is no longer true with the addition of size. Please update the markdown to reflect this change. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/289/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>